zellij-server/src/screen.rs

```diff
@@ -1530,6 +1535,77 @@ impl Screen {
         }
     }
 
+    pub fn move_active_tab_to_left(&mut self, client_id: ClientId) -> Result<()> {
+        if self.tabs.len() < 2 {
+            info!("cannot move tab to left: only one tab exists");
```

Comment:
I think it would be better to do it with `debug`.

Reply:
Done :+1: